### ISSUE TITLE
Add CI checks

### ISSUE THREAD

**Issue by simonge** [eic/afterburner#11]
Currently checks on the output of the afterburner are done manually and on an adhoc sample of events which will not represent the full range of event generators and settings which need afterburning.

I would like to have some checks set up to make sure any changes made do not inadvertently break a current workflow.

**Comment by DraTeots** [comment]
That is absolutely right idea. The caveat is that there at the last time we run the validation some changes were needed on pythia side. I think Brian needs to be involved to resurrect and finish this part.

**Comment by simonge** [comment]
Initially I was hoping to just making sure the output of the afterburner is consistent with the previous output, or changes as expected. Automatically reproducing the validation plots against pythia would probably come second. Although I guess the current version should be compared to pythia in case anything has changed in the meantime.

Trying to do this by myself is going to be very inefficient, the whole CI workflow definitions in yml is currently rather opaque to me and the versions in other repositories, necessarily, look very complicated.

**Comment by DraTeots** [comment]
It might be an off topic, but recently ChatGPT [provided me](https://github.com/JeffersonLab/ccdb/blob/v2-main/.github/workflows/ci.yaml) with pretty good starting .yaml file for github actions that was working out of the box.

**Comment by simonge** [comment]
Thanks, definitely not off topic. I can't pretend I wasn't trying to get Copilot to start me off but probably asking the wrong things.

**Comment by DraTeots** [comment]
[That is what I was asking](https://chat.openai.com/share/f41c0dc3-d9fa-4a7a-ad47-cae7d2714dca)
